network-config-analyzer/CalicoPolicyYamlParser.py

```diff
@@ -569,24 +569,25 @@ def _set_affects_ingress_egress(self, policy_spec, is_profile, res_policy):
         if not res_policy.affects_egress and 'egress' in policy_spec:
             self.syntax_error('A NetworkPolicy with egress field but no "Egress" in its policyTypes', policy_spec)
 
-    def _set_selected_peers(self, policy_spec, is_profile, res_policy):
+    def _set_selected_peers(self, policy_spec, is_profile, policy_name):
```

Comment:
change function name to `_get_selected_peers` and also update docstring description (it does not "set" the selected peers member of the policy, but rather returns a set of the selected peers).